sdk/core/core/inc/az_result.h

```diff
@@ -63,6 +63,9 @@ typedef enum az_result {
   AZ_ERROR_JSON_INVALID_STATE = AZ_MAKE_ERROR(AZ_JSON_FACILITY, 2),
   AZ_ERROR_JSON_STACK_OVERFLOW = AZ_MAKE_ERROR(AZ_JSON_FACILITY, 3),
 
+  AZ_ERROR_HTTP_PIPELINE_INVALID_POLICY = AZ_MAKE_ERROR(AZ_CORE_FACILITY, 5),
```

Comment:
I think it should be in AZ_HTTP_FACILITY

Reply:
moved